src/main/java/com/getindata/connectors/http/internal/table/sink/HttpDynamicSink.java

```diff
@@ -113,15 +122,18 @@ public SinkRuntimeProvider getSinkRuntimeProvider(Context context) {
 
         var insertMethod = tableOptions.get(INSERT_METHOD);
         var contentType = getContentTypeFromFormat(tableOptions.get(FactoryUtil.FORMAT));
+
+        // TODO ESP-98 add headers to DDL and add tests for this
         HttpSinkBuilder<RowData> builder = HttpSink
             .<RowData>builder()
             .setEndpointUrl(tableOptions.get(URL))
             .setSinkHttpClientBuilder(JavaNetSinkHttpClient::new)
             .setElementConverter((rowData, _context) -> new HttpSinkRequestEntry(
                 insertMethod,
-                contentType,
                 serializationSchema.serialize(rowData)
-            ));
+            ))
+            .setProperty(CONTENT_TYPE_HEADER, contentType)
+            .setProperties(properties);
```

Comment:
Got your point,.
I would suggest removing `formatContentTypeMap` and relay on DDL configuration elementally. 
Even now it may happen that "applciation/json" would not be expected needed etc, and endpoint will drop the request.
And we will not be able to change it nicely, since are always adding content-type.

There may be scenarios were content type will not be needed.
So i guess format should not dictate the http-header. Maybe in future we will come up with some other cleaver enhancement. 

I would suggest to drop the map and simply relay on DDL configuration.

Reply:
I'm not against it, we may drop the map. I just wanted to have as simple configuration as we could get, and having to write
```
format = 'json',
header = 'application/json'
```
seemed a little cumbersome and/or redundant to me. However, we can have it explicit, I don't mind :)

Reply:
But what is so special about "application/json" or content type header to give it it dedicated short name?
It is no more special than other HTTP headers and in my opinion there is no reason to threat it differently than other headers.

It is not a required header by any stnadard, all depends on the endpoint configuration

Reply:
Fair point, you're right :) We can drop it 👍